src/app/seamly2d/dialogs/export_layout_dialog.cpp

```diff
@@ -763,7 +763,7 @@ void ExportLayoutDialog::writeSettings() const
  */
 QString ExportLayoutDialog::modeString() const
 {
-    QString modeStr = QStringLiteral();
+    QString modeStr = QStringLiteral("");
```

Comment:
Why this unrelated change here?

Reply:
I think  the compiler was complaining... the current Qt 5 is using QString modeStr = QStringLiteral(""); 

I must have made the change sometime between the Qt6 branch and now.

Reply:
I found that the change was commited on the "develop" branch on [this commit](https://github.com/FashionFreedom/Seamly2D/commit/36da4c1d42ce236f48735305eb834690287e9305#diff-35adbcb349615d8ee5f0e5be7a20c76220e61b1b78c2288b820cea2e04d1e278)

I will omit this change from the PR because it only produces a clang warning and doesn't crash the program